assets/blocks/course-outline/index.js

```diff
@@ -0,0 +1,3 @@
+import './course-outline-block';
```

Comment:
I didn't want to break it in folders yet, so we can decide it while the feature grows. But probably I think we'll want to create at least a folder for each of these 3 blocks and add the related components in the respective folder.